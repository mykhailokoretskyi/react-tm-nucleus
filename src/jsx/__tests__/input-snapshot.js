import React from 'react';
import Input from '../input';
import renderer from 'react-test-renderer';

describe('Input', () => {
    it('mocks match snapshots', () => {
        const component = renderer.create(
            <Input type="text" placeholder="enter here..." />
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
