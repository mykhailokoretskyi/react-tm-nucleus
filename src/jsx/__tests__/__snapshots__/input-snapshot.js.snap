exports[`Input mocks match snapshots 1`] = `
<input
  checked={false}
  disabled={false}
  onBlur={[Function bound _onBlur]}
  onChange={[Function bound _onChange]}
  onClick={[Function bound _onClick]}
  onFocus={[Function bound _onFocus]}
  onMouseEnter={[Function bound _onMouseEnter]}
  onMouseLeave={[Function bound _onMouseLeave]}
  placeholder="enter here..."
  required={false}
  type="text"
  value="" />
`;
